app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
in our lecture we used `before_action :authenticate_user!` here to restrict every page by default. Can you add this too?